NimbleExtension/Sources/JSONMapper/Models/JSONAPIError.swift

```diff
@@ -0,0 +1,27 @@
+//  JSONAPIError.swift
+//
+//  Created by Pirush Prechathavanich on 4/25/18.
+//  Copyright © 2018 Nimbl3. All rights reserved.
+//
+
+import Foundation
+
+struct JSONAPIError: Error, Decodable, Equatable {
```

Comment:
I just think that `NimbleExtension` don't need this because it depends on the API right?

Reply:
Do I need any document to understand about all models?

Reply:
I think it is needed. @teamdisc can you comment?